src/targets/mobile/config.xml

```diff
@@ -112,4 +112,5 @@
     <plugin name="cordova-blur-app-privacy-screen" spec="https://github.com/lifeofcoding/cordova-blur-app-privacy-screen.git" />
     <plugin name="cordova-universal-links-plugin" spec="https://github.com/Mailbutler/cordova-universal-links-plugin" />
     <plugin name="cordova-plugin-network-information" spec="^2.0.1" />
+    <plugin name="cordova-plugin-ionic-keyboard" spec="^2.2.0" />
```

Comment:
I don't know exactly if this plugin provides a shrink method or not but you should test the login on iOS and Android to see if nothing breaks (I remember that cozy-authentication has a few conditions on Keyboard.X)

Reply:
The plugin does not have the shrinkView method of cozy-authentication. Do you have somewhere the reasons why we had to remove the cordova-plugin-keyboard ? I've searched the PR and found https://github.com/cozy/cozy-banks/pull/1311. This PR uninstalls the plugin but I still see the Keyboard being checked in cozy-authentication. I wonder if we still have something to do on the subject ?